.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: bash
+
+script:
+  - ./licenseSorter.py && git diff --exit-code
```

Comment:
can  we add this: https://github.com/Alfresco/alfresco-build-tools/blob/master/.travis.pre-commit.yml

and have pre-commit execute on build too?